src/popup.js

```diff
@@ -0,0 +1,115 @@
+import { postComment, getComment } from './api.js';
+
+const updateCount = (comments) => {
+  const title = document.getElementById('comment-title');
+  if (comments === undefined) {
+    title.textContent = 'Comments (0)';
+  } else {
+    title.textContent = `Comments (${comments})`;
+  }
+};
+const getComments = async (itemID) => {
+  const title = document.getElementById('comment-title');
+  const commentList = document.querySelector('.comment-container');
+  let refreshNewComment = await getComment(itemID);
+  try {
+    if (Array.isArray(refreshNewComment)) {
+      commentList.innerHTML = '';
+      const numberOfComments = refreshNewComment.length;
+      refreshNewComment.forEach((rnc) => {
+        const newCommentx = document.createElement('li');
+        newCommentx.className = 'oneComment';
+        newCommentx.innerHTML = `
+        <p class="text-gray-400 text-xs">${rnc.creation_date} - ${rnc.username}</p>
+        <p class="text-white">${rnc.comment}</p>`;
+        commentList.appendChild(newCommentx);
+        if (numberOfComments > 0) {
+          updateCount(numberOfComments);
+        } else {
+          title.textContent = 'Comments (0)';
+        }
+      });
+    } else {
+      title.textContent = 'Comments (0)';
+    }
+  } catch (error) {
+    refreshNewComment = [];
+  }
+};
+
+const createComment = (movieElementId) => {
+  const form = document.getElementById('comment-form');
+  const formName = document.getElementById('name');
+  const formComment = document.getElementById('comments');
+  form.addEventListener('submit', async (event) => {
+    event.preventDefault();
+    const createDate = new Date();
+    const newName = formName.value;
+    const newComment = formComment.value;
+    const commentObj = {
+      item_id: movieElementId,
+      username: newName,
+      comment: newComment,
+      creation_date: createDate,
+    };
+    formName.value = '';
+    formComment.value = '';
+    await postComment(commentObj);
+    getComments(movieElementId);
+  });
+};
+
+const popupWins = (root, title, img, genre, status, premiered, lang, movieElementId) => {
+  if (root.querySelector('.popup-container')) { root.lastElementChild.remove(); }
+  document.body.classList.add('overflow-hidden');
+  const overlay = document.createElement('div');
+  overlay.className = 'fixed top-0 left-0 w-screen h-screen bg-black bg-opacity-80 z-20';
+  const modal = document.createElement('div');
+  modal.className = 'popup-container fixed h-11/12 w-11/12 md:h-5/6 md:w-1/2 z-30 bg-gray-800 rounded-lg p-8 top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 flex flex-col';
+
+  modal.innerHTML = `
+      <div>
+        <div class="modal-top">
```

Comment:
- [ ] It will be better to add the following property to `modal-top` class, so anyone using your app can submit the comment.
```css
    max-height: 76vh;
    overflow-y: scroll;
```
> Your design
![image](https://github.com/redmarverick/MMDB/assets/53402600/23d2d941-32a2-4250-aba9-2bd5a29118f3)